src/de/darmstadt/tu/crossing/CipherInputStream.cryptsl

```diff
@@ -0,0 +1,30 @@
+SPEC javax.crypto.CipherInputStream
+OBJECTS
+    byte[] b;
+    int off;
+    int len;
+    long n;
```

Comment:
You never use n, as far as I can see.

Reply:
Oh yeah, I was using "n" as input for another method which I didn't declare afterward in the EVENTS section